client/src/main.jsx

```diff
@@ -1,10 +1,15 @@
-import React from "react";
 import ReactDOM from "react-dom/client";
+import { Provider } from 'react-redux';
+import store from './Redux/store';
 import App from "./App.jsx";
 import "./index.css";
+import { ErrorBoundary } from "react-error-boundary";
 
 ReactDOM.createRoot(document.getElementById("root")).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+  <Provider store={store}>
+ <ErrorBoundary fallback={<h1>Something went wrong</h1>}>
```

Comment:
better to add error component, then in the future if we will want to edit/style it - it will be easier

Reply:
added ErrorFallBack components